game-core/src/main/java/games/strategy/engine/lobby/client/ui/LobbyGameTableModel.java

```diff
@@ -1,25 +1,13 @@
 package games.strategy.engine.lobby.client.ui;
```

Comment:
A more heavy refactor was done to this class, drops dependencies on Messengers altogether and simplified the test, which in turn also no longer depended on messengers.